### ISSUE TITLE
[autocomplete] not working on 6.2RC2 (okay on RC1)

### ISSUE THREAD

**Issue by tacman** [symfony/ux#572]
I updated to Symfony 6.2RC2 and got this error, I revered to RC1 and it was okay again.

I haven't researched it, just reporting it since Symfony 6.2 is so close to release.  Even reinstalling ux-autocomplete didn't fix it.  But perhaps purging the cache would have worked.

 [KO]
Script cache:clear returned with error code 1
!!  
!!   // Clearing the cache for the dev environment with debug true                  
!!  
!!  
!!  In FileLoader.php line 178:
!!                                                                                 
!!    Unable to find file "@AutocompleteBundle/Resources/routes.php" in @Autocomp  
!!    leteBundle/Resources/routes.php (which is being imported from "/home/tac/g/  
!!    survos/jardin/config/routes/ux_autocomplete.yaml"). Make sure the "Autocomp  
!!    leteBundle" bundle is correctly registered and loaded in the application ke  
!!    rnel class. If the bundle is registered, make sure the bundle path "@Autoco  
!!    mpleteBundle/Resources/routes.php" is not empty.                             
!!                                                                                 
!!  
!!  In Kernel.php line 233:
!!                                                                     
!!    Unable to find file "@AutocompleteBundle/Resources/routes.php".  
!!                                                                     
!!  
!!  cache:clear [--no-warmup] [--no-optional-warmers]
!!  
!!  
Script @auto-scripts was called via post-update-cmd

**Comment by zavarock** [comment]
Just change it to 
```yaml
ux_autocomplete:
    resource: '@AutocompleteBundle/config/routes.php'
```
in **config/routes/ux_autocomplete.yaml**

Ref: https://github.com/symfony/ux/releases/tag/v2.6.0

**Comment by weaverryan** [comment]
Yup thanks! Also running `composer recipes:update` will do this for you - a final BC break before we released 2.6.0 without experimental status.
